### PR TITLE
Added recipe for paramiko

### PR DESCRIPTION
**Pull request by pmlandwehr**
Closes https://github.com/conda-forge/staged-recipes/issues/1106

Currently building `1.17.2`, which is an old version but required for building `Fabric3`. We can update once this build is completed.

### CODE REVIEW

**Comment by jakirkham** [comment]
Already adding it in PR ( https://github.com/conda-forge/staged-recipes/pull/1156 ). Would you like me to add you as a contributor? Also we are trying to handle some of its dependencies if you would like to help. 😉

**Comment by jakirkham** [comment]
Sorry missed you were building an old version. Let me close mine out and we will go with yours. Please feel free to pull anything from that PR that you need.

File: recipes/paramiko/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  license: LGPL 2.1
+  license_file: LICENSE
+  summary: 'SSH2 protocol library'
+  license_family: Public-Domain
+  dev_url: https://github.com/paramiko/paramiko/
+  doc_url: http://docs.paramiko.org/
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/paramiko/meta.yaml]: Please add @vamega based on our [discussion]( https://github.com/conda-forge/staged-recipes/pull/1156/files#r75807235 ) and myself.

**Comment by pmlandwehr** [comment]
Seems to have gotten weirdly munged into a build of `cryptography-vectors`. Closing and reopening...

**Comment by jakirkham** [comment]
Sorry shoved a bunch of stuff through staged-recipes. May need a rebase or merge with `master` to clean it up.

File: recipes/paramiko/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/paramiko/meta.yaml]: Done below.

File: recipes/paramiko/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - paramiko
+
+about:
+  home: http://www.paramiko.org/
+  home: https://github.com/paramiko/paramiko/
+  license: LGPL 2.1
+  license_file: LICENSE
+  summary: 'SSH2 protocol library'
+  license_family: Public-Domain
```
Comment (jakirkham) [inline recipes/paramiko/meta.yaml]: Hmm...not sure this is accurate. Maybe GPL? Not really sure what they do with this.

**Comment by pmlandwehr** [comment]
Appveyor build seems to have succeeded but gotten stuck. Closing/reopening.

**Comment by jakirkham** [comment]
One thing still to address.

**Comment by pmlandwehr** [comment]
@jakirkham should be done.

**Comment by jakirkham** [comment]
Thanks for doing this, @pmlandwehr.

**Comment by jakirkham** [comment]
> Seems to have gotten weirdly munged into a build of `cryptography-vectors`. Closing and reopening...

Also we have made some changes so that this should be avoided in the future. See PR ( https://github.com/conda-forge/staged-recipes/pull/1373 ) for the bulk of the change and PR ( https://github.com/conda-forge/conda-forge-webservices/pull/53 ) for changes to the linter. Basically merged recipes that haven't converted shouldn't show up in one's PR.

**Pull request merged**